Library/Homebrew/extend/os/mac/hardware/cpu.rb

```diff
@@ -107,6 +107,8 @@ def arm_family
           :arm_lightning_thunder
         when 0x573b5eec, 0x1b588bb3 # ARMv8.4-A (Firestorm, Icestorm)
           :arm_firestorm_icestorm
+        when 0xda33d83d             # ARMv8.4-A (Blizzard, Avalanche)
```

Comment:
I think the Apple A15 is actually ARMv8.5-A.

Reply:
Do you have any hard info? The only info I found is this WP page, which states otherwise: https://en.wikipedia.org/wiki/Comparison_of_ARMv8-A_cores

Reply:
Nothing official, but:

https://nanoreview.net/en/soc/apple-a15-bionic
https://www.headshotreviews.com/apple-a15-bionic-chip-apl1w05-specification/
https://screenrant.com/snapdragon-8-gen-1-vs-a15-bionic-qualcomm-apple-chips-compared/